Heartland-iOS-SDK.xcodeproj/project.pbxproj

```diff
@@ -7,8 +7,14 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+<<<<<<< HEAD
+		143A4379298AB2980027BC3F /* String+ProjectName.swift in Sources */ = {isa = PBXBuildFile; fileRef = 143A4378298AB2980027BC3F /* String+ProjectName.swift */; };
+=======
+		DD2409302991688C00597A23 /* HpsUpaStartCardResponse.swift in Sources */ = {isa = PBXBuildFile; fileRef = DD24092F2991688C00597A23 /* HpsUpaStartCardResponse.swift */; };
+>>>>>>> master
```

Comment:
@felixhps We can no longer view the source files in Xcode, think it's because of this conflict.

cc @wdesimini